packages/shared_preferences/shared_preferences_android/android/src/main/kotlin/io/flutter/plugins/sharedpreferences/SharedPreferencesPlugin.kt

```diff
@@ -250,25 +249,17 @@ class SharedPreferencesPlugin() : FlutterPlugin, SharedPreferencesAsyncApi {
   /** Class that provides tools for encoding and decoding List<String> to String and back. */
   class ListEncoder : SharedPreferencesListEncoder {
     override fun encode(list: List<String>): String {
-      try {
```

Comment:
These try/catch blocks weren't doing anything since they were only catching and throwing the same types, so I removed them as an opportunistic cleanup.